english/shrunk/shrunken.md

```diff
@@ -0,0 +1,7 @@
+All of a sudden, you regret sneaking that taste of chocolate. The room around you feels unstable, and then seems to be growing at a rapid pace. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.